lib/payment_provider/handlers/async_handler.rb

```diff
@@ -13,6 +13,7 @@ class AsyncHandler
       def call(event)
         ::Rails::logger.info("WEBHOOK: #{event.type} CONNECT: #{event.try(:account)} LIVEMODE: #{event.livemode}")
         handler = HANDLER_IMPLS[event.type]
+        Rollbar.info('webhook', event)
```

Comment:
Does Rollbar use delayed job? If not, could we defer to that unless this is already async (which, given it's name is probably the case...)